src/getting-started.md

```diff
@@ -192,6 +192,14 @@ sometimes be long. PRs are never merged by hand.
 [r+]: https://github.com/rust-lang/rust/pull/78133#issuecomment-712726339
 [bors]: https://bors.rust-lang.org/queue/rust
 
+### Reviewing PRs
+
+Another way to contribute to the project is by reviewing existing PRs. We have a [review checklist]
+you can use to get started. We suggest reviewing the PRs of existing contributors. Remember that
```

Comment:
I don't think that suggesting reviewing the PRs of existing contributors is essential here; funneling beginners to help each other is very useful to get the easy nits out of the way quickly.